http://lists.parisc-linux.org/pipermail/parisc-linux/2007-January/031142.html
http://sourceware.org/ml/binutils/2007-01/msg00045.html

2007-01-27  Mike Frysinger  <vapier@gentoo.org>

	* elf32-hppa.c [elf32_bed]: Define for hpux, linux, and netbsd.
	[INCLUDED_TARGET_FILE]: Remove.

--- binutils-2.17.50.0.10/bfd/elf32-hppa.c
+++ binutils-2.17.50.0.10/bfd/elf32-hppa.c
@@ -4656,6 +4656,7 @@
 #define ELF_MACHINE_CODE	EM_PARISC
 #define ELF_MAXPAGESIZE		0x1000
 #define ELF_OSABI		ELFOSABI_HPUX
+#define elf32_bed		elf32_hppa_hpux_bed
 
 #include "elf32-target.h"
 
@@ -4665,8 +4666,9 @@
 #define TARGET_BIG_NAME			"elf32-hppa-linux"
 #undef ELF_OSABI
 #define ELF_OSABI			ELFOSABI_LINUX
+#undef elf32_bed
+#define elf32_bed		elf32_hppa_linux_bed
 
-#define INCLUDED_TARGET_FILE 1
 #include "elf32-target.h"
 
 #undef TARGET_BIG_SYM
@@ -4675,5 +4677,7 @@
 #define TARGET_BIG_NAME			"elf32-hppa-netbsd"
 #undef ELF_OSABI
 #define ELF_OSABI			ELFOSABI_NETBSD
+#undef elf32_bed
+#define elf32_bed		elf32_hppa_netbsd_bed
 
 #include "elf32-target.h"
